docs/sources/installation/kubernetes.md

```diff
@@ -0,0 +1,238 @@
+```
+### Create Grafana Enterprise Kubernetes manifest
+Create a file called `grafana.yaml` and paste the following contents in. This yaml is identical to the yaml for the OS install except for the additional references to the Configmap that contains your Grafana config file and the Secret that contains your license.
+
+```yaml
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: grafana
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 1Gi
+  storageClassName: local-path
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: grafana
+  name: grafana
+spec:
+  selector:
+    matchLabels:
+      app: grafana
+  template:
+    metadata:
+      labels:
+        app: grafana
+    spec:
+      containers:
+        - image: grafana/grafana-enterprise:latest
+          imagePullPolicy: IfNotPresent
+          name: grafana
+          ports:
+            - containerPort: 3000
+              name: http-grafana
+              protocol: TCP
+          readinessProbe:
+            failureThreshold: 3
+            httpGet:
+              path: /robots.txt
+              port: 3000
+              scheme: HTTP
+            initialDelaySeconds: 10
+            periodSeconds: 30
+            successThreshold: 1
+            timeoutSeconds: 2
+          resources:
+            limits:
+              memory: 4Gi
+            requests:
+              cpu: 100m
+              memory: 2Gi
+          volumeMounts:
+            - mountPath: /var/lib/grafana
+              name: grafana
+            - mountPath: /etc/grafana
+              name: ge-config
+            - mountPath: /etc/grafana/license
+              name: ge-license
+      volumes:
+        - name: grafana
+          persistentVolumeClaim:
+            claimName: grafana
+        - name: ge-config
+          configMap:
+            name: ge-config
+        - name: ge-license
+          secret:
+            secretName: ge-license
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: grafana
+spec:
+  ports:
+    - port: 3000
+      protocol: TCP
+      targetPort: http-grafana
+  selector:
+    app: grafana
+  sessionAffinity: None
+  type: LoadBalancer
+```
+ 
+### Send manifest to Kubernetes API Server
+
+Run the following command to create the necessary resources in your cluster. 
+```bash
+kubectl apply -f grafana.yaml
+```
+
+### Check that it worked
+Run the following: 
+```bash
+kubectl port-forward service/grafana 3000:3000
+```
+Now if you navigate to `localhost:3000` in your browser, you should see a Grafana login page. Use `admin` for both the username and password values to login.
+
+A successful Grafana Enterprise installation with a valid license should show an opened lock icon when you mouse over the shield icon in the left menu bar. You can also verify by scrolling down to the bottom of the page.
+![lock_ge](../../lock_ge.png "Showing left menu")
```

Comment:
I'd like to add 2 images in here, but not sure where I'm supposed to put them. This attempt I was doing was not working (I was trying to put them in grafana/docs) and I don't think its likely the right place. I see some images in grafana/public/img but when I tried putting the images in there, I still had the same path problems that the images wouldn't render.

Reply:
@09jvilla, @osg-grafana or I can show you where to put the images. It's kind of weird, and mentioned in the [Grafana docs folder README](https://github.com/grafana/grafana/tree/master/docs#add-images).